components/zcash_protocol/src/consensus.rs

```diff
@@ -661,7 +661,7 @@ impl From<BranchId> for u32 {
             BranchId::Nu5 => 0xc2d6_d0b4,
             BranchId::Nu6 => 0xc8e7_1055,
             #[cfg(zcash_unstable = "nu7")]
-            BranchId::Nu7 => 0xffff_ffff,
+            BranchId::Nu7 => 0x7719_0ad8,
```

Comment:
This value is not yet established by a deployment ZIP, and so MUST NOT be set to any non-0xffff_ffff value.
```suggestion
            BranchId::Nu7 => 0xffff_ffff,
```